apps/nrm/views/content/session-timeout.md

```diff
@@ -2,4 +2,4 @@
 
 Sorry, your session has timed out due to inactivity. Any information you may have entered has been deleted.
 
-To continue to use this service, you will need to <a href="/start">start again</a>.
+To continue to use this service, you will need to <a class="govuk-link" href="/start">start again</a>.
```

Comment:
Would it be more readable to use md syntax in this file and just extend any necessary govuk classes via the css?

Reply:
Replaced the <a> tag with md syntax.